Tests/LibJS/test-js.cpp

```diff
@@ -58,6 +58,42 @@ TESTJS_GLOBAL_FUNCTION(get_weak_map_size, getWeakMapSize)
     return JS::Value(weak_map->values().size());
 }
 
+TESTJS_GLOBAL_FUNCTION(force_gc, forceGC)
+{
+    if (!vm.argument(0).is_string()) {
+        vm.throw_exception<JS::TypeError>(global_object, JS::ErrorType::NotA, "String");
+        return {};
+    }
+
+    auto& variable_name = vm.argument(0).as_string();
+    if (vm.exception())
+        return {};
+
+    // In native functions we don't have a lexical environment so get the outer via the execution stack.
+    auto outer_environment = vm.execution_context_stack().last_matching([&](auto& execution_context) {
+        return execution_context->lexical_environment != nullptr;
+    });
+    if (!outer_environment.has_value())
+        return {};
```

Comment:
We need to throw an exception here, or else you'll be returning an empty value to javascript land, which is usually pretty crashy